README.md

```diff
@@ -1 +1,139 @@
-# Stock-Metrics-WebApp
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Stock-Metrics-WebApp</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Stock-Metrics-WebApp <a name="Stock-Metrics-WebApp"></a>
+
+Companies is a React and Redux SPA application that displays interesting information about various companies in the world. It fetchs all the information that it displays using API request to a server.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React
+- Redux
+- Git, Github
+- VSCode
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Git,
+> Github,
+> React
+> Redux
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/react/">React</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Basic set up
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/Ashnaali3255/Stock-Metrics-WebApp.git
+```
+
+### Prerequisites
+
+- NodeJS
+- React & Redux
+- Browser
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your dist folder:
+
+git clone: https://github.com/Ashnaali3255/Stock-Metrics-WebApp.git
+
+### Install
+
+npm install
+
+### Usage
+
+Use this project in the browsers of desktops and mobile devices
+
+### Run tests
+
+npm run test
+
+### Deployment
+
+You can deploy this project on GitHub Pages
```

Comment:
- [ ] To make your README more descriptive and professional, I think it would be better to update your `Getting Started` section with correct and appropriate information to each sub-section such as information of how to `access the cloned` directory, `run build for productioon` bundles, `use/run` your application,... **To help you to fill the above information, you can just refer [this Gist file](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96).**